### PR TITLE
basic responsive nav

### PR DESCRIPTION
**Pull request by binoyskumar92**
This pull request addresses issue #64

**Summary of changes**

- Added a hamburger icon and change sin navbar based on screen size

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
+                    <Hamburger
+                        onToggle={() => setHamburgerPressed(!hamburgerButtonPressed)}></Hamburger>
+                </div>
+                <div
+                    className={`${
+                        hamburgerButtonPressed ? 'flex flex-col' : 'hidden'
+                    } lg:flex lg:flex-row w-screen  bg-gray-50 sm:p-4 lg:justify-between items-center gap-y-4 z-10 border-b-2  border-gray-200 font-montserrat font-medium`}>
+                    <div className='w-20 h-8 flex align-middle'>
+                        <NavLink exact to={routes.home.path}>
+                            <img src='/logo.png' alt='logo'></img>
```
Comment (victorlin) [inline src/components/Navbar.tsx:27]: +1 to @ababaian's comment in Slack, this logo is redundant

Comment (binoyskumar92) [reply src/components/Navbar.tsx:27]: fixed it

File: src/components/Navbar.tsx [hunk 2]
```diff
@@ -1,61 +1,75 @@
...
+                        </NavLink>
+                    </div>
+                    <Hamburger
+                        onToggle={() => setHamburgerPressed(!hamburgerButtonPressed)}></Hamburger>
+                </div>
+                <div
+                    className={`${
+                        hamburgerButtonPressed ? 'flex flex-col' : 'hidden'
+                    } lg:flex lg:flex-row w-screen  bg-gray-50 sm:p-4 lg:justify-between items-center gap-y-4 z-10 border-b-2  border-gray-200 font-montserrat font-medium`}>
```
Comment (victorlin) [inline src/components/Navbar.tsx:24]:
```suggestion
                    } lg:flex lg:flex-row w-screen  bg-gray-50 sm:p-2 lg:justify-between items-center gap-y-4 z-10 border-b-2  border-gray-200 font-montserrat font-medium`}>
```

The `sm:p-4` creates some unnecessary height now that the image defines the height (not sure why). On my screen, `sm:p-2` brings the nav bar closer to the original height.

(compare https://serratus.io to https://add-responsive-navbar.serratus.io)

**Review by victorlin — CHANGES_REQUESTED** [review]
One more minor thing, but otherwise looks good!

File: src/components/Navbar.tsx (hunk 2 above)
Comment (victorlin) [reply src/components/Navbar.tsx]: Applied here: https://github.com/serratus-bio/serratus.io/pull/157/commits/75ad389cb878d025e4698d11658405913a83ff27

**Review by victorlin — APPROVED** [review]

**Pull request merged**